src/setup/devnet/index.js

```diff
@@ -561,8 +561,17 @@ export class Devnet {
                     ];
 
                     // create heimdall folder
-                    shell.exec(`sudo mkdir -p /var/lib/heimdall`)
-                    shell.exec(`sudo chmod 777 -R /var/lib/heimdall/`)
+                    for (let i = 0; i < this.totalNodes; i++) {
+                        await execa('ssh', [
+                            `-o`, `StrictHostKeyChecking=no`, `-o`, `UserKnownHostsFile=/dev/null`,
```

Comment:
have you tested this? Got some issues in the past when using `execa ssh` with `sudo`

Reply:
I didn't face any issue while testing it. Is there a specific scenario that comes to your mind that I should look for ?